tests/e2eTestForBoost/step-definitions/boost.steps.js

```diff
@@ -1,39 +1,31 @@
 const {Given, When, Then} = require('@cucumber/cucumber')
 let helpers = require('../../../helpers/common-actions');
-const assert = require('assert');
 
 
-Given(/^Login with (.*) and (.*)$/, async (username, password) => {
-   await boostPostPage.performLogin(username, password)
+Given(/^I login with (.*) and (.*)$/, async (username, password) => {
+   await boostPostPage.login(username, password)
 })
-
-Then (/^Create a text post$/,async function (){
-    await myFeedPage.CreatePost()
-    await helpers.waitForTimeout(page,5000);
-    await  myFeedPage.CreatePostFunc();
+Then(/^I create a text post$/,async function (){
+    await myFeedPage.createPost();
+    await helpers.waitForTimeout(page,10000);
```

Comment:
I tried without **wait** here but the tool is very fast so I often encounter "Unable to create post" often when I run 2-3 times so I reduced the wait to 5000 ms to avoid flaky tests.